sdk/template/template-dpg/src/api/WidgetServiceContext.ts

```diff
@@ -11,7 +11,8 @@
 
 import { ClientOptions } from "@azure-rest/core-client";
 import { TokenCredential, isTokenCredential } from "@azure/core-auth";
-import getClient, { WidgetServiceContext } from "../rest/index.js";
+import { WidgetServiceContext } from "../rest/index.js";
+import getClient from "../rest/widgetServiceClient.js";
```

Comment:
why is this not imported from the index module as well?

Reply:
This import is derived from the import statement in this custom layer file. 

https://github.com/Azure/azure-sdk-for-js/blob/84b130d7543f04d723b85ce01f7f84df7bdbc7ee/sdk/template/template-dpg/sources/customizations/src/api/WidgetServiceContext.ts#L5C5-L5C5

Reply:
More generally, unless the user declares an identical export in the custom layer, custom files can only import customized symbols from the file in which they're declared. It'd be possible to track down the generated import statement and emit the generated module specifier in cases where that module doesn't exist in the custom layer, but I don't think it's worth the investment.